src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -948,16 +956,34 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 
 	// MARK: - Test Type Selection
 	
-	/// Shows the Type of Tests screen
-	/// - Parameter isSelfTestTypePreselected: If the process was started via self-report with self-test on the Manage Your Tests View, the Self-test entry shall be pre-selected. Otherwise, no entry shall be selected.
-	private func showSRSTestTypeSelectionScreen(isSelfTestTypePreselected: Bool = false) {
-		let srsTestTypeSelectionViewController = SRSTestTypeSelectionViewController(
-			viewModel: SRSTestTypeSelectionViewModel(isSelfTestTypePreselected: isSelfTestTypePreselected),
+	/// Checks the SRS Flow prerequisites to continue with SRS Test Type Selection.
+	/// Shows a specific error alert, if at least on prerequisite has fault.
+	/// - Parameters:
+	/// 	- srsFlowType: The SRS Flow Type the user has selected
+	/// 	- isLoading: The callback that should be execute while fetching the self service parameters from app configuration.
+	private func checkSRSPrerequisitesToContinueSRSFlow(
```

Comment:
Prerequisites + to + XYZ 

```suggestion
	private func checkSRSPrerequisitesToSRSFlow(
```

Reply:
good, renamed